### PR TITLE
empty-card-module: #1 - add empty card component

### PR DESCRIPTION
**Pull request by Esha-Rajbhandari**
### empty-card-module: #1 - add empty card component

Capture

### CODE REVIEW

File: src/components/achievements/Achievements.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-import EditOptions from '~/components/editoptions/EditOptions';
 import { Add } from '~/assets/image';
+import React, {useState} from 'react';
+import EmptyCard from '../emptycard/EmptyCard';
+import CardHeader from '~/components/cardheader/CardHeader';
 import CardFooter from '~/components/cardfooter/CardFooter';
+import EditOptions from '~/components/editoptions/EditOptions';
 
 const Achievements = () => {
+	const [isDataAvailabel, setData] = useState(true);
```
Comment (elizaRjb) [inline src/components/achievements/Achievements.js:9]: I guess you meant to write `isDataAvailable` instead of `isDataAvailabel`?

Comment (Esha-Rajbhandari) [reply src/components/achievements/Achievements.js]: 🆗

**Comment by sagunji** [comment]
@Esha-Rajbhandari You have merge conflicts here.

File: src/components/ButtonHandler/ButtonHandler.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const modalBtnHandler = e => {
+  e.preventDefault();
+  setModal(!showModel);
```
Comment (sagunji) [inline src/components/ButtonHandler/ButtonHandler.js:3]: did you mean `showModal`?

Comment (Esha-Rajbhandari) [reply src/components/ButtonHandler/ButtonHandler.js:3]: 🆗

**Review by sagunji — APPROVED** [review]

**Pull request merged**